guides/common/modules/proc_loading-the-default-openscap-contents.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Loading_the_Default_OpenSCAP_Contents_{context}"]
+= Loading the Default OpenSCAP Contents
+
+Use this procedure to load the default OpenSCAP contents on {ProjectServer}.
+This procedure ensures that the data streams from the *SCAP Security Guide* are the loaded and assigned to all organizations and locations.
+
+The *SCAP Security Guide* is provided by the {ProjectServer}'s operating system and installed in `/usr/share/xml/scap/ssg/content/`.
+Note that the available data streams depend on the operating system version on which {Project} runs.
+On {EL} 8, the package provides the SCAP contents only for {EL} 6{range}8.
+For information on acquiring the SCAP content for {EL} 9, see xref:Uploading_Additional_SCAP_Content_{context}[].
+
+.Prerequisites
+* Your user account has the `create_scap_contents` permission.
+
+.Procedure
+* Use the following Hammer command on {ProjectServer}:
```

Comment:
Same as above. Maybe I'll have a go at it myself to reword some instructions to run Hammer CLI commands.